integration-tests/simple-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -27,6 +27,15 @@
         <url-pattern>/servlet</url-pattern>
     </servlet-mapping>
 
+    <servlet-mapping>
+        <!--
+        allows testing servlet mapping with wildcard, can make servlet path to be empty/null according to spec
+        but in practice this behavior relies on app server implementation and might differ.
+        -->
+        <servlet-name>TestServlet</servlet-name>
+        <url-pattern>/test-servlet/*</url-pattern>
```

Comment:
Does this reproduce the exact issue we're seeing on Weblogic?

Reply:
Not directly, if I understand the servlet spec correctly this should in theory make `servletPath` to be empty, but in practice with Tomcat this is not the case. for example we get `servletPath` = `/test-servlet` and `pathInfo` = `/info` when request URL is `/test-servlet/info` which is also a bit confusing. I am not sure it's really worth adding an integration test for that case though, I've kept it mostly for manual testing.